src/app/components/ConsensusTransactionMethod/index.tsx

```diff
@@ -103,7 +103,6 @@ const MethodIconContent: FC<MethodIconContentProps> = ({
         <Typography
           sx={{
             fontWeight: 'inherit',
-            textTransform: 'capitalize',
```

Comment:
will add this to Consensus MVP ticket because now we have capitalize names in runtimes, but not in consensus. (Iconography Figma issue)